src/concrete/ml/quantization/quantized_module.py

```diff
@@ -99,28 +99,39 @@ def __init__(
         # Set base attributes for API consistency. This could be avoided if an abstract base class
         # is created for both Concrete ML models and QuantizedModule
         # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/2899
```

Comment:
better to move this fixme right above `self.input_quantizers` because it talks about these attributes, no the `self.ordered_module_input_names` and similar